included bootsrap

tessa5 opened the pull request:
added a little bit more than usual but it still does not look good... Diego,

tessa5 committed:
included bootsrap

added a little bit more than usual but it still does not look good... Diego,

do-diegoortiz commented on css/main.css:
```diff
@@ -2,7 +2,7 @@
 a { text-decoration:none;}
 .clear {clear: both;}
-body {
+/*body {
```
Do not comment out. If you don't need it anymore you should delete it.

do-diegoortiz commented on index.html:
```diff
@@ -1,81 +1,113 @@
+     <link rel="stylesheet" href="css/main.css"> 
+    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js" integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1" crossorigin="anonymous"></script>
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js" integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script>
```
Why did you include the js part of bootstrap?

tessa5 replied:
i did that because the dropdowns cannot work without the files

do-diegoortiz replied:
Good job

do-diegoortiz commented on index.html:
```diff
@@ -1,81 +1,113 @@
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js" integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1" crossorigin="anonymous"></script>
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js" integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script>
+    <style>
+    .navbar, .jumbotron {
```
Styles must be in the style.css or any file .css

do-diegoortiz commented on index.html:
```diff
@@ -1,81 +1,113 @@
\ No newline at end of file
+    </div>
+</body>
+</html>
```
Add an empty line at the end of the file

do-diegoortiz reviewed (commented):
Please, attach a screenshot of the outcome in the PR description.

tessa5 committed:
organized

made the changes- put style in the css file and deleted the styles that were commented out

tessa5 reviewed (commented):
![snip](https://user-images.githubusercontent.com/51070829/60202035-73db4180-9852-11e9-9894-b976942a0ba2.JPG)
i

do-diegoortiz reviewed (approved):
Good job, you can merge it

tessa5 committed:
colorful nav

do-diegoortiz commented on css/main.css:
```diff
@@ -5,6 +5,25 @@ a { text-decoration:none;}
     border-radius: 0;
+}
+.navbar {
+    background-color:crimson;
```
You should try to not modify the classes from bootstrap. If you want a special style, you should create a new class.

do-diegoortiz reviewed (approved):
It's ok. Merge this pull request and delete the branch.

tessa5 committed:
updated profile

did an update on my portfolio and added some few files, made some changes but i have some errors in .js
